packages/core/src/controllers/SiweController.ts

```diff
@@ -0,0 +1,87 @@
+import { subscribeKey as subKey } from 'valtio/utils'
```

Comment:
Not sure what's going on here, but this file name is `SiweController` while everywhere in the code it is `SIWEController`

Reply:
I updated the casing for consistency, but I think git doesn't take the change into account if you just update the casing of the file name. Will fix it 👍